node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -530,7 +531,10 @@ void doConverge(NodeAgentContext context) {
                 }
 
                 aclMaintainer.ifPresent(maintainer -> maintainer.converge(context));
-                if (runOrdinaryWireguardTasks) wireguardTasks.forEach(task -> task.converge(context));
+                if (runOrdinaryWireguardTasks) {
+                    Optional<Container> finalContainer = container;
```

Comment:
This seems redundant, as the name does not add any value(?)

Reply:
Unfortunately, Java requires variables referenced in lambdas to be effectively final :-(